README.md

```diff
@@ -1,6 +1,80 @@
-ASP.NET Core IIS Integration
-========
-
-This repo hosts the ASP.NET Core middleware for IIS integration.
+This repo hosts the ASP.NET Core middleware for IIS integration and the AspNetCoreModule
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+=======
+# ASP.NET Core Module
+
+The ASP.NET Core Module is an IIS Module which is responsible for process
+management of ASP.NET Core http listeners and to proxy requests to the process
+that it manages.
+
+## Installing the latest ASP.NET Core Module
+The ASP.NET Core Module for IIS can be installed on servers without installing the .NET Core runtime. You can download the [Windows (Server Hosting) installer](https://go.microsoft.com/fwlink/?linkid=832756) and run the following command from an Administrator command prompt:
+``DotNetCore.1.1.0.Preview1-WindowsHosting.exe OPT_INSTALL_LTS_REDIST=0 OPT_INSTALL_FTS_REDIST=0``
```

Comment:
This is outdated. Should we just remove it?

Reply:
I'm going to drop the ANCM readme for now. @shirhatti can you update before 2.1?